File: AngularPeople/src/app/shared/models/person.ts
export class Person{
    constructor(
        public id:string,
        public firstName:string,
        public lastName:string,
        public email:string,
        public phone:string,
        public personTypeId:string
    ){}
}

export class PersonType{
    constructor(
        public id:string,
        public description:string,
    ){}
}

File: AzureWebServices/Core.Services/EFModels/Person.cs
using System;
using System.Collections.Generic;

namespace Core.Services.EFModels
{
    public partial class Person
    {
        public Guid Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public Guid? PersonTypeId { get; set; }

        public virtual PersonType PersonType { get; set; }
    }
}
